.github/workflows/local-testnet.yml

```diff
@@ -26,7 +26,16 @@ jobs:
             repo-token: ${{ secrets.GITHUB_TOKEN }}
       - name: Install ganache
         run: npm install ganache@latest --global
-
+      - name: Install geth
```

Comment:
maybe good to mention in setup.md https://github.com/sigp/lighthouse/blob/unstable/book/src/setup.md?plain=1#L12

Reply:
Hey @atokama. We are using this for testing for 4844 specific stuff. This might not get merged for quite some time to the main branch. We have another PR https://github.com/sigp/lighthouse/pull/3807 where we should be ideally adding this and updating the docs for building post merge local testnets. Haven't been able to get around to that yet though :sweat_smile: